sql/rename.go

```diff
@@ -136,3 +136,57 @@ func (p *planner) RenameTable(n *parser.RenameTable) (planNode, error) {
 
 	return &valuesNode{}, nil
 }
+
+// RenameIndex renames the index.
+// Privileges: WRITE on database.
```

Comment:
I see you made the code change, but forgot to update the comment. It should be `Privileges: WRITE on table`